pybullet build time: Jan  9 2023 19:56:43
b3Warning[examples/Importers/ImportURDFDemo/BulletUrdfImporter.cpp,126]:
expected a world element, continuing without it.b3Warning[examples/Importers/ImportURDFDemo/BulletUrdfImporter.cpp,126]:
No links found in URDF fileTraceback (most recent call last):
  File "/Users/han/Desktop/mybots/simulate.py", line 7, in <module>
    simulation = SIMULATION(directOrGui, solutionID)
                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/han/Desktop/mybots/simulation.py", line 23, in __init__
    self.robot = ROBOT(self.solutionID, self.world)
                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/han/Desktop/mybots/robot.py", line 17, in __init__
    self.robotId = p.loadURDF("body" + self.solutionID + ".urdf")
                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
pybullet.error: Cannot load URDF file.
